README.md

```diff
@@ -235,15 +240,39 @@ Bump new version number in:
 And commit these changes:
 
 ```bash
+$ git checkout master
 $ git add .
-$ git commit -m "Bump version 1.16.8"
+$ git commit -m "Bump version 1.17.0"
 $ git push origin master
 ```
 
+### (optional) Cherry Pick Commits Included in the Release
+
+You may want to cherry pick the commits to include in the release. You first
+need to get the commit ID of the previous release (let's call it `base commit
+ID`). Then create a new branch and cherry pick all the commits you need, from
+the oldest to the most recent. Use the commit ID of the merge of every pull
+request you want to include in your release. Don't forget to also cherry pick
+the commit for the version bump.
+
+```bash
+git checkout <base commit ID>
+git checkout -b v1.17.1
+git cherry-pick -m 1 <commit ID number 1>
+git cherry-pick -m 1 <commit ID number 2>
+...
+git cherry-pick -m 1 <commit ID number X>
+git push --set-upstream origin v1.17.0
```

Comment:
isn't it 1.17.1?

Reply:
No the README usually contains the current version number (i.e. 1.17.0). Hence when I release a new version I just type `:%s/1.17.0/1.17.1/g` and it's updated everywhere in the README.